client/influxdb.go

```diff
@@ -44,7 +44,7 @@ func NewClient(c Config) (*Client, error) {
 	return &client, nil
 }
 
-func (c *Client) Query(q Query) (influxdb.Results, error) {
+func (c *Client) Query(q Query) (*influxdb.Results, error) {
```

Comment:
Reference types usually aren't returned as pointers.  E.g., http://golang.org/pkg/strings/#Split returns `[]string` not `*[]string`.